site/app/libraries/database/DatabaseQueries.php

```diff
@@ -9372,6 +9399,45 @@ public function checkNonMappedUsers() {
         return $this->rowsToArray($this->submitty_db->rows());
     }
 
+    /**
+     * Adds a component grader to the active_graders table
+     * @param Component $component
+     * @param bool $isTeam
+     * @param string $grader_id
+     * @param string $graded_id
+     * @return void
```

Comment:
There's no need to add PHPDoc annotations if they don't provide additional information beyond that provided by the native PHP annotations.
```suggestion
```